spec/mailers/club_application_mailer_spec.rb

```diff
@@ -32,7 +32,7 @@
 
     it { should have_subject 'Application Confirmation' }
     it { should deliver_to application.mail_address.format }
-    it { should deliver_from 'Zach Latta <team@hackedu.us>' }
+    it { should deliver_from 'Zach Latta <team@hackclub.io>' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.